.github/workflows/check-combinations.yml

```diff
@@ -0,0 +1,47 @@
+name: "Check combinations.json"
+on:
+  schedule:
+  # Run at midnight on the first of every month
+  # https://crontab.guru/once-a-month
+  - cron: "0 0 1 * *"
+
+  push:
```

Comment:
I'd like to use  `workflow_dispatch:` instead of `push` 
It allow admin to run when it needed. It is convenient for test purpose.

Reply:
It is also worth considering
```
on:
  push:
     paths:
       - 'ci/**'
```

Reply:
Those are really good ideas! I will do that!